superdesk/errors.py

```diff
@@ -558,5 +558,25 @@ def httpPushError(cls, exception=None, destination=None):
         return PublishHTTPPushError(14000, exception, destination)
 
 
+class PublishHTTPPushClientError(PublishHTTPPushError):
+    _codes = {
+        14001: "HTTP push publish client error",
+    }
+
+    @classmethod
+    def httpPushError(cls, exception=None, destination=None):
+        return PublishHTTPPushClientError(14001, exception, destination)
+
+
+class PublishHTTPPushServerError(PublishHTTPPushError):
+    _codes = {
+        14002: "HTTP push publish server error",
+    }
+
+    @classmethod
```

Comment:
Yeah, I'm stuck having to pass a code as a first argument though to a SuperdeskPublishError class, so creating a static method to hold this code like the other classes was the best I think for now. (Also all these args these notifiers are looking for in the base class.